styles/burger.css

```diff
@@ -0,0 +1,183 @@
+/* Burger menu*/
+.quiz-main-menu {
+  display: flex;
+  position: fixed;
+  width: 225px;
+  height: 100vh;
+  top: 0;
+  left: -225px;
+  box-sizing: border-box;
+  background: rgba(0, 0, 0, 0.4);
```

Comment:
The background could more opaque or even solid. Currently the way the description and the button labels overlap make it a little hard to read.